oak_functions/client/java/src/com/google/oak/functions/client/AttestationClient.java

```diff
@@ -123,46 +123,25 @@ public void attest(ManagedChannel channel, Predicate<ConfigurationInfo> verifier
       throw new NullPointerException("Channel must not be null.");
     }
     this.channel = channel;
-    StreamingSessionStub stub = StreamingSessionGrpc.newStub(channel);
+    stub = UnarySessionGrpc.newBlockingStub(channel);
 
-    // Create server response handler.
-    messageQueue = new ArrayBlockingQueue<>(1);
-    StreamObserver<StreamingResponse> responseObserver = new StreamObserver<StreamingResponse>() {
-      @Override
-      public void onNext(StreamingResponse response) {
-        try {
-          messageQueue.put(response);
-        } catch (Exception e) {
-          if (e instanceof InterruptedException) {
-            Thread.currentThread().interrupt();
-          }
-          logger.log(Level.WARNING, "Couldn't send server response to the message queue: " + e);
-        }
-      }
-
-      @Override
-      public void onError(Throwable t) {
-        Status status = Status.fromThrowable(t);
-        logger.log(Level.WARNING, "Couldn't receive response: " + status);
-      }
-
-      @Override
-      public void onCompleted() {}
-    };
-    requestObserver = stub.stream(responseObserver);
```

Comment:
This is pretty nice, using unary requests allows us to use the blocking gRPC api for calls. This means we no longer have to implement our own blocking queue to wait for server responses